fuse/data/ops/ops_debug.py

```diff
@@ -0,0 +1,110 @@
+from typing import Any, Hashable, List, Sequence, Optional
+from fuse.data.utils.sample import get_sample_id
+from fuse.utils import NDict
+from fuse.data import OpBase
+import numpy
+import torch
+
+
+class OpDebugBase(OpBase):
+    """Base class for debug operations"""
+
+    def __init__(self, sample_ids: Optional[List[Hashable]] = None, first_sample_only: bool = False):
+        """
+        :param sample_ids: apply for the specified sample ids. To apply for all set to None.
+        :param first_sample_only: apply for the first sample only
+        """
+        super().__init__()
+        self._sample_ids = sample_ids
+        self._first_sample_only = first_sample_only
+        self._first_sample_done = False
+
+    def should_debug_sample(self, sample_dict: NDict) -> bool:
```

Comment:
Suggestion:
Added an option to limit the amount of samples to a number (and not only by names) in case the user want to debug 5 random samples (without mentioning the names).
I can see it useful for the `OpVisImage`.  (In case the user want to examine 3-5 random images in a single run).

Reply:
Good idea! I will change `first_sample_only` to `num_samples_to_debug`